### PR TITLE
merge penile fibromatosis

### LINKED ISSUE #7142: MONDO:0006361 & MONDO:0008231 Peyronie's disease [Merge]
**Mondo term (ID and Label)**
MONDO:0006361 penile fibromatosis

**Reason for deprecation**
These two terms share synonyms and X-ref/match to same NCIT term. UMLS treats them as synonyms on C0030848

**Term to be merged with**
MONDO:0008231 Peyronie's disease
**Your nano-attribution (ORCID)**
If you don't have an ORCID, you can sign up for one [here](https://orcid.org/)

### PR DESCRIPTION
**Pull request by twhetzel**
closes #7142

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -160885,25 +160870,37 @@ is_a: MONDO:0003847 {source="https://orcid.org/0000-0001-5208-3432"} ! hereditar
 id: MONDO:0008231
 name: Peyronie disease
 def: "A condition characterized by hardening of the penis due to the formation of fibrous plaques on the dorsolateral aspect of the penis, usually involving the membrane (tunica albuginea) surrounding the erectile tissue (corpus cavernosum penis). This may eventually cause a painful deformity of the shaft or constriction of the urethra, or both." [MESH:D010411]
+subset: inferred_rare
 subset: orphanet_rare {source="Orphanet:2870"}
 subset: rare
 synonym: "Induratio penis plastica" EXACT [DOID:8616, Orphanet:2870]
+synonym: "penile fibromatosis" EXACT [MONDO:0006361, NCIT:C3316]
+synonym: "penile induration" EXACT [NCIT:C3316]
+synonym: "penis fibromatosis" EXACT [MONDO:patterns/location]
```
Comment (twhetzel) [inline src/ontology/mondo-edit.obo:160879]: I don't know what `MONDO:patterns/location` is. Should this be removed?

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -160885,25 +160870,37 @@ is_a: MONDO:0003847 {source="https://orcid.org/0000-0001-5208-3432"} ! hereditar
 id: MONDO:0008231
 name: Peyronie disease
 def: "A condition characterized by hardening of the penis due to the formation of fibrous plaques on the dorsolateral aspect of the penis, usually involving the membrane (tunica albuginea) surrounding the erectile tissue (corpus cavernosum penis). This may eventually cause a painful deformity of the shaft or constriction of the urethra, or both." [MESH:D010411]
+subset: inferred_rare
```
Comment (twhetzel) [inline src/ontology/mondo-edit.obo:160873]: Should this `subset` be removed?

**Comment by twhetzel** [comment]
The qc failed this test "qc-exact-synonyms-non-exact-mappings.sparql". I'm not sure what change needs to be made to have this test pass. Any help is appreciated.

File: src/ontology/mondo-edit.obo [hunk 3]
```diff
@@ -160885,25 +160870,37 @@ is_a: MONDO:0003847 {source="https://orcid.org/0000-0001-5208-3432"} ! hereditar
...
+synonym: "Peyronie's fibromatosis" EXACT [DOID:8616, NCIT:C3316]
 xref: DOID:8616 {source="MONDO:equivalentTo"}
+xref: EFO:1000466 {source="MONDO:equivalentTo"}
 xref: ICD10CM:N48.6 {source="DOID:8616"}
 xref: ICD9:607.85 {source="DOID:8616"}
 xref: MESH:D010411 {source="DOID:8616"}
-xref: NCIT:C3316 {source="MONDO:relatedTo", source="DOID:8616"}
+xref: NCIT:C3316 {source="MONDO:equivalentTo", source="EFO:1000466"}
+xref: NCIT:C3316 {source="MONDO:equivalentTo", source="DOID:8616"}
 xref: OMIM:171000 {source="MONDO:equivalentTo", source="DOID:8616"}
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:160890]: These 2 x-refs are the same, ie these are duplicated. The sources should be grouped together and we should keep only one of them.

File: src/ontology/mondo-edit.obo [hunk 4]
```diff
@@ -160885,25 +160870,37 @@ is_a: MONDO:0003847 {source="https://orcid.org/0000-0001-5208-3432"} ! hereditar
...
 xref: Orphanet:2870 {source="MONDO:equivalentObsolete"}
 xref: SCTID:1335005 {source="DOID:8616"}
 xref: SCTID:155931002 {source="DOID:8616"}
 xref: SCTID:198028006 {source="DOID:8616"}
 xref: UMLS:C0030848 {source="MONDO:equivalentTo", source="DOID:8616", source="MONDO:ncbi_mim2gene_medline", source="OMIM:171000"}
+xref: UMLS:C0030848 {source="MONDO:relatedTo", source="NCIT:C3316"}
 is_a: MONDO:0002036 {source="DOID:8616"} ! penile disorder
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:160897]: This UMLS x-ref is duplicated. Also, it is both "equivalent" and "related". 
We need to figure out which one it is, and keep one.

**Review by sabrinatoro — CHANGES_REQUESTED** [review]
@twhetzel 
There is 2 comments about duplicated x-refs.

File: src/ontology/mondo-edit.obo (hunk 3 above)
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:160890]: updated

File: src/ontology/mondo-edit.obo (hunk 4 above)
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:160897]: After reviewing values for the `UMLS:C0030848` it should be "MONDO:equivalentTo".

**Review by sabrinatoro — APPROVED** [review]
It all looks good. Thank you @twhetzel !

**Pull request merged**